charts/caraml-authz/templates/service-account.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.serviceAccount.create }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+    name: {{ default "mlp-authz" .Values.serviceAccount.name }}
```

Comment:
Let's call it `caraml-authz` for consistency with the chart name

Reply:
Changed.